src/kata1.fizzBuzz.js

```diff
@@ -1,5 +1,17 @@
 const fizzBuzz = (number) => {
+    if (number % 5 === 0 && number % 3 === 0) {
+        return 'FizzBuzz'
+    }
 
-}
+    if (number % 5 === 0) {
+        return 'Buzz'
+    }
+
+    if (number % 3 === 0) {
+        return 'Fizz'
+    }
+
```

Comment:
Instead of 3 separate if statements, one else if or switch statement would have been more concise here. Code works though and passes the tests.